eng/jacoco-test-coverage/pom.xml

```diff
@@ -444,172 +444,172 @@
     <dependency>
       <groupId>com.azure.spring</groupId>
       <artifactId>spring-cloud-azure-core</artifactId>
-      <version>4.3.0-beta.1</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-core;current} -->
+      <version>4.3.0</version> <!-- {x-version-update;com.azure.spring:spring-cloud-azure-core;current} -->
```

Comment:
There is a version the sdk/spring/README.md should also be updated, why I don't see it in this PR? Did it show up when you run the upgrade version script?

Reply:
The readme won't be updated because it uses the `dependency` version instead of the `current` version.